aptos-move/framework/aptos-framework/sources/configs/gas_schedule.move

```diff
@@ -0,0 +1,55 @@
+/// This module defines structs and methods to initialize VM configurations,
+/// including different costs of running the VM.
+module aptos_framework::gas_schedule {
+    use std::error;
+    use std::string::String;
+
+    use aptos_framework::reconfiguration;
+    use aptos_framework::system_addresses;
+    use aptos_framework::timestamp;
+    use aptos_framework::util::from_bytes;
+
+    /// Error with config
+    const ECONFIG: u64 = 0;
+    /// The provided gas constants were inconsistent.
+    const EGAS_CONSTANT_INCONSISTENCY: u64 = 1;
+
+    struct GasEntry has store, copy, drop {
+        key: String,
+        val: u64,
+    }
+
+    struct GasSchedule has key, copy, drop {
+        entries: vector<GasEntry>
+    }
+
+    public fun initialize(account: &signer, gas_schedule_blob: vector<u8>) {
+        timestamp::assert_genesis();
+        system_addresses::assert_aptos_framework(account);
+
+        assert!(
+            !exists<GasSchedule>(@aptos_framework),
+            error::already_exists(ECONFIG)
+        );
+
+        // TODO(Gas): check if gas schedule is consistent
+
+        let gas_schedule: GasSchedule = from_bytes(gas_schedule_blob);
+
+        move_to(account, gas_schedule);
+    }
+
+    public entry fun set_gas_schedule(account: &signer, gas_schedule_blob: vector<u8>) acquires GasSchedule {
```

Comment:
This doesn't need to be an entry function as it's only callable as part of the governance process from a script anyway. Scripts can call non-entry public functions

Reply:
Actually it looks we'll need this entry point at least before the new governance stuff come out. A smoke test currently uses this to set the gas schedule.

Reply:
Makes sense